karate-core/src/main/java/com/intuit/karate/report/ReportUtils.java

```diff
@@ -154,6 +154,49 @@ private static Throwable appendSteps(List<StepResult> steps, StringBuilder sb) {
         return error;
     }
 
+    private static Element addCustomTags(Element testCase, Document doc, ScenarioResult sr){
+        //Adding requirement and test tags
+        Element properties = null;   
+        String custom_tags = System.getProperty("custom_tags");         
```

Comment:
thanks ! first please edit the PR to target the `develop` branch.

what are these system properties ? is this some X-Ray thing `custom_tags`

Reply:
Edited to target the `develop` branch.

I wanted this change to be as generic as possible to serve Xray purpose but also to be able to be reused for other tools. As such it is possible to define the tags that will be used in the Scenarios and the correspondent tags that will appear in the Junit XML report. For that in the @BeforeAll or @BeforeClass we can define what we want:
    ```
@BeforeClass
    public static void beforeClass() {
        System.setProperty("custom_tags", "test, requirement");
        System.setProperty("custom_xml_tags", "test_key, requirement");
    }
```

We must use `custom_tags` and `custom_xml_tags` that is what this code change is searching for. 

If this definition is set and match the tags added in the Scenario it will add the `properties`  element to the Junit XML report, if not nothing will be added.

Reply:
@CMCunha to be very honest I don't like this. karate snapshots the system.env when a suite starts (and I'm not sure what the life cycle is if junit 5 etc) and it is immutable. any config in my opinion should be driven via the cli `Main` or the `Runner.Builder`. so I vote for removing this. if this is some accepted industry standard, we can add a method similar to `outputCucumberJson()` but right now I'm not convinced

Reply:
Thanks you for your feedback @ptrthomas .
The other options you suggested were adding n option in the cli `Main` (like you suggested) in a specific new option or by the `-D` entry that means that every time I want to execute tests I need to add those options to the execution command, that doesn't seem an easy option for the user. 

Or adding an option in the `Runner.Builder` that will enable and configure this feature, like adding a method `custom_tags()`, did I get the right idea?

I saw that there is a config file in Karate, I tried to add configuration values there but were unable to get those in the `ReportUtils.java` class, do you think it is possible?

Reply:
@CMCunha how about we just release this without this system property thing. I was curious if this is documented somewhere as a standard e.g the `custom_tags` name etc.

and after users try it and really feel / explain why they need some extra configuration, we can consider. I don't want to waste time on this if no-one needs it

Reply:
Ok, I removed the system property. Now all tags that follow format `@key=value` will be added to a properties element in XML report.